examples/python/compression/README.md

```diff
@@ -0,0 +1,31 @@
+## Compression with gRPC Python
+
+gRPC offers lossless compression options in order to decrease the number of bits
+transferred over the wire. Three levels of compression are available:
+
+ - `grpc.NoCompression` - No compression is applied to the payload. (default)
+ - `grpc.Deflate` - The "Deflate" algorithm is applied to the payload.
+ - `grpc.Gzip` - The Gzip algorithm is applied to the payload.
+
+See [the gRPC Compression Spec](https://github.com/grpc/grpc/blob/master/doc/compression.md)
+for more information.
```

Comment:
The compression argument is quite clean and easy to use. Do you think a short snippet show how to set it will make it more convenient  for readers?